### DISCUSSION TITLE
what is the format of the .model file generated after training MACE model

### DISCUSSION THREAD

**Discussion by mofeilu** [ACEsuit/mace#422] [Q&A] [answered]
I understand that after run a training script, a .model file will be generated which store the trained model.  Just curious what is the format of this .model file?  is it self defined by MACE or is it based on torch I/O formats?  thanks!

**Comment by stargolike** [comment]
i think it has full model information, and if you need to run by lammps, you need transform this file.

**Comment by ilyes319** [comment, marked as answer]
It uses the standard pickle of torch, and can be used with torch.load. As mentionned, in order to load in lammps, you need to convert it to a jit format. See the documentation for futher instructions. https://mace-docs.readthedocs.io/en/latest/guide/lammps.html
